src/services/specification.service.ts

```diff
@@ -45,6 +47,12 @@ export class SpecificationService {
         return asyncApiDoc;
       })
       .catch(err => {
+        try {
+          const asyncapiSpec = YAML.load(rawSpec);
+          MonacoService.updateLanguageConfig((asyncapiSpec as { asyncapi: string }).asyncapi);
```

Comment:
Can't we do something like this?

```suggestion
          const asyncapiSpec: { asyncapi: string } = YAML.load(rawSpec);
          MonacoService.updateLanguageConfig(asyncapiSpec.asyncapi);
```

Or like this if that fails:

```suggestion
          const asyncapiSpec = YAML.load(rawSpec) as { asyncapi: string };
          MonacoService.updateLanguageConfig(asyncapiSpec.asyncapi);
```